extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -66,7 +66,7 @@ require([
 
 		sourcePoint.initDetection();
 
-		if (context.opts.pageFairDetection) {
+		if (pageFair.canBeInitialized(context)) {
 			pageFair.initDetection(context);
```

Comment:
you are doing the check twice, one check is here and the second one (for the same variable) is inside initDetection. Consider simplification.